project/src/main/webapp/script.js

```diff
@@ -127,24 +127,28 @@ function createTrendElement(trend, val) {
 
 /** Displays trend frequency bubbles on page. */
 function getTrendBubbles(content) {
-  const bubbleContainer = document.getElementById('trend-frequency-data');
+  const bubbleFirstRow = document.getElementById('frequency-row-1');
+  const bubbleSecondRow = document.getElementById('frequency-row-2');
   const bubbleSizes = getTrendBubbleSize(content);
+  let length = (bubbleSizes.size)/2;
+  let i = 0;
   for (const [key,value] of bubbleSizes.entries()) {  
-    bubbleContainer.appendChild(createTrendBubbles(key,bubbleSizes.get(key)));
+    i < length ? bubbleFirstRow.appendChild(createTrendBubbles(key,bubbleSizes.get(key),i)) : bubbleSecondRow.appendChild(createTrendBubbles(key,bubbleSizes.get(key),i));
+    i += 1;
   } 
 }
 
 /** Creates an element that represents a trend and its frequency. */
-function createTrendBubbles(trend, size) {
+function createTrendBubbles(trend, size,i) {
```

Comment:
nits: space between size and i + since this is creating a single bubble, maybe rename to createTrendBubble